functions-home.php

```diff
@@ -0,0 +1,457 @@
+<?php
+/**
+ * Homepage functions
+ */
+
+function hide_editor_from_homepage() {
```

Comment:
Maybe we can look at this together. I tried to add this as a separate function, but doesn't work by itself.